apps/aehttp/src/aehttp_dispatch_int.erl

```diff
@@ -16,8 +16,7 @@ handle_request('PostSpendTx', #{'SpendTx' := SpendTxObj}, _Context) ->
             #{<<"recipient_pubkey">> := EncodedRecipientPubkey,
               <<"amount">>           := Amount,
               <<"fee">>              := Fee} = SpendTxObj,
-            case aec_base58c:safe_decode(
-                   account_pubkey, EncodedRecipientPubkey) of
+            case aens:resolve(account_pubkey, EncodedRecipientPubkey) of
```

Comment:
May be worth an integration of the description in the corresponding field in swagger.yaml